ivy/functional/frontends/numpy/linalg/solving_equations_and_inverting_matrices.py

```diff
@@ -18,3 +18,11 @@ def inv(a):
 # pinv
 def pinv(a, rtol=1e-15, hermitian=False):
     return ivy.pinv(a, rtol)
+    
+    
+# tensorinv
+def tensorinv(a, b, reverse=False, ind=2):
+    ret = ivy.tensorinv(a, ind=2)
```

Comment:
Hello @ShiftLove, does calling this work? Ivy has no `tensorinv` function as far as i know. this will throw an error.

Reply:
Hi @CatB1t it actually works without an error